h/models/user.py

```diff
@@ -342,9 +342,7 @@ def __acl__(self):
         # may update the user
         user_update_principal = "client_authority:{}".format(self.authority)
 
-        # Are these the same thing or not?
         terms.append((security.Allow, user_update_principal, Permission.User.UPDATE))
-        terms.append((security.Allow, user_update_principal, Permission.Profile.UPDATE))
```

Comment:
The `User.UPDATE` is granted to any client authority which matches the user authority, the `Profile.UPDATE` is only granted to the user to edit themselves.